server/lib/configuration.js

```diff
@@ -439,6 +439,11 @@ var conf = module.exports = convict({
       doc: 'Sentry API key',
       format: String
     },
+    api_secret: {
```

Comment:
not fxa-dev, but puppet config. PR here : https://github.com/mozilla-services/puppet-config/pull/1984